.gitignore

```diff
@@ -22,3 +22,10 @@ hs_err_pid*
 target
 .gradle
 build
+gradlew
```

Comment:
you both added this to the .gitignore file but still kept the file in the repo. Make sure any files in the .gitignore are deleted from the history

Reply:
Hi @sztupy , I was not sure actually about these files, didn't know which ignoring or not...

Reply:
We can skip those files, they are not really mandatory, but if we add them to the .gitignore we also need to make sure they are not present in the repository, otherwise `.gitignore` will be ignored

Reply:
I saw they're actually present in Master so I just reverted those changes and committed the example... Thanks! @sztupy